internal/domain/file_content_response.go

```diff
@@ -0,0 +1,9 @@
+package domain
+
+type FileContentResponse struct {
+	Filename  string `json:"filename"`
+	Content   string `json:"content"`
```

Comment:
Include JSON struct tags for 'Position' to ensure consistent serialization and deserialization behavior, as this can affect how the data is consumed by APIs.